pcsx2/Common.h

```diff
@@ -20,6 +20,10 @@
 static const u32 BIAS = 2;				// Bus is half of the actual ps2 speed
 static const u32 PS2CLK = 294912000;	//hz	/* 294.912 mhz */
 
+extern int psxEEmultiplier; // This variable controls and set's the Multiplier value for the EE to Iop cycle clock, 
```

Comment:
I prefer if the comments are above the psxEEmultiplier, will look much more cleaner.